code/modules/admin/secrets.dm

```diff
@@ -379,6 +380,31 @@
 			for(var/obj/machinery/light/L in machines)
 				L.broken()
 
+		if("anime")
+			if(!check_rights(R_FUN))
+				return
+			feedback_inc("admin_secrets_fun_used",1)
+			feedback_add_details("admin_secrets_fun_used","CC")
+			message_admins("[key_name_admin(usr)] made everything kawaii.")
+			for(var/mob/living/carbon/human/H in mob_list)
+				H << sound('sound/AI/animes.ogg')
+
+				if(H.dna.species.id == "human")
```

Comment:
There really isn't, purrbation was just too lazy to add a check to see if a mutant race already had a different tail/set of ears so blacklisted other races instead.

Also I guess "LORE", but who the fuck cares about that catgirls are

N O T
C A N O N